addon/webextension/background/main.js

```diff
@@ -84,113 +69,12 @@ this.main = (function() {
     });
   }
 
-  function shouldOpenMyShots(url) {
-    return /^about:(?:newtab|blank)/i.test(url) || /^resource:\/\/activity-streams\//i.test(url);
-  }
-
-  browser.browserAction.onClicked.addListener(catcher.watchFunction((tab) => {
```

Comment:
In lazy-background I changed this to a function, `main.onClicked = ...` – then I just changed how it was invoked. I think the same would be appropriate here.